src/js/appbar.js

```diff
@@ -2,7 +2,7 @@
     Wood.AppBar = Marionette.LayoutView.extend({
       tagName: "appbar",
       attributes: {
-        class: 'appbar',
+        class: 'wood appbar',
```

Comment:
you like this better than:

    <div class="wood">
         <!-- create any wood plugins -->
         <appbar class="appbar">
    </div>

also, any reason to use `appbar` as the tag and class?